src/test/java/net/obvj/performetrics/PerformetricsTestDrive.java

```diff
@@ -38,16 +39,20 @@ private static void testStopwatch1() throws InterruptedException, IOException
         Stopwatch sw = Stopwatch.createStarted();
         test(sw);
 
-        System.out.println("\n\n--------------------------------------\n");
-        System.out.println("Resuming the same stopwatch...\n");
-        sw.start();
-        test(sw);
+        for (int i = 0; i < 4; i++)
+        {
+            System.out.println("\nResuming the same stopwatch...\n");
+            sw.start();
+            test(sw);
+        }
     }
 
     private static void test(Stopwatch sw) throws InterruptedException, IOException
     {
         // Enforcing some wall-clock time...
-        Thread.sleep(2000);
+
+        int sleepTimeMillis = new Random(987).nextInt(3000);
```

Comment:
Replace with predictable variable times (e.g.: 500, 1000, 1500, 2000, 2500) to secure that the sequence is not lost.